packages/vue-ds/src/components/Pagination/Pagination.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+  <section v-if="!isInvalid" :class="PaginationClass">
+    <div :class="PaginationContainerClass">
+      <button
+        v-if="hasPreviousCondition"
+        class="paginationButton"
+        :disabled="isLoading"
+        @click="prevPage"
+      >
+        <AngleLeft :width="12" :height="23" :viewBox="'-1 3 12 7'" />
```

Comment:
how you are passing `string` here, we do not need to use v-bind here. `viewBox="-1 3 12 7"` should works as well ;)

Reply:
hehehe i'm sorry 😞 I'll do it 😄